routes/api-v0.1/club-route.js

```diff
@@ -57,6 +58,45 @@ var ClubRoute = (function (){
             response.json('400', {'message': 'Bad Request'});
         }
     };
+
+    ClubRoute.prototype.removeClub = function(request,response){
+        var club_id = request.params.club_id;
+        ClubModel.remove({_id: club_id}, function(err, doc){
+           if(err){
+               response.send({error: err});
+            } else {
+               response.send(doc);
```

Comment:
The variable doc is a response of the operation and indicates if the process of removing was completed successfully. But we cannot do something in front-end side with this information. It will be better if you response the record id that the ORM has been removed from the database.